rpm/common/Makefile

```diff
@@ -0,0 +1,40 @@
+NAME          = archivematica-common
+VERSION       = 1.4.1
+RPM_TOPDIR    = "/rpmbuild"
+DOCKER_VOLUME = "/src"
+DOCKER_IMAGE  = "rpmbuild-$(NAME)-$(VERSION)"
```

Comment:
Can you use `DOCKER_VOLUME = "/src"? I've been doing that in the other packages.